database/src/migrations/release.0.22/smoketest_release.sql

```diff
@@ -147,14 +157,14 @@ begin
   insert into submission_status (occurrence_submission_id, submission_status_type_id, event_timestamp) values (_occurrence_submission_id, (select submission_status_type_id from submission_status_type where name = 'Rejected'), now()) returning submission_status_id into _submission_status_id;
   insert into submission_message (submission_status_id, submission_message_type_id, event_timestamp, message) values (_submission_status_id, (select submission_message_type_id from submission_message_type where name = 'Notice'), now(), 'A notice message at stage published.');
   select count(1) into _count from submission_status;
-  assert _count = 4, 'FAIL submission_status';
+  assert _count = 5, 'FAIL submission_status';
   select count(1) into _count from submission_message;
   assert _count = 4, 'FAIL submission_message';  
 
-  raise notice 'survey status (project_id, survey_id, survey_status):';
-  for _survey_status_rec in execute _survey_status_query loop
-    raise notice 'survey status results are % % %', _survey_status_rec.project_id, _survey_status_rec.survey_id, _survey_status_rec.survey_status;
-  end loop;
+--  raise notice 'survey status (project_id, survey_id, survey_status):';
+--  for _survey_status_rec in execute _survey_status_query loop
+--    raise notice 'survey status results are % % % %', _survey_status_rec.project_id, _survey_status_rec.survey_id,  _survey_status_rec.occurrence_id, _survey_status_rec.survey_status;
+--  end loop;
```

Comment:
can these comments be removed or are they required?

Reply:
good question. this isn't really part of the product but a testing/debugging tool. i may be the only one using it at the moment for smoke testing the database and loading data for further examination. should it simply be removed from the release?

Reply:
well the reason I was wondering is cause it was commented out - so unless the plan is to add it back later, it can probably just be removed?